dev/prefixer-plugin.js

```diff
@@ -0,0 +1,57 @@
+import webpack from 'webpack';
+import {escapeRegExp} from '../src/utils/regex.js';
+
+const {Compilation, sources} = webpack;
+
+export default class PrefixerPlugin {
+  constructor(options = {}) {
+    if (options.replaceClassnamePrefixRegex == null) {
```

Comment:
this ins't an `Regex`, so should we call these `oldClassNamePrefix` and `newClassNamePrefix`?